cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -311,10 +312,7 @@ public void performRunLevelStop(HttpRequest request, HttpResponder responder,
                                   @QueryParam("graceful") String gracefulShutdownSecs,
                                   @QueryParam("version") String version) throws Exception {
```

Comment:
Could we remove this `"version"` param that was wrongly added?

Reply:
good point.  will remove